docs/source/admin/traffic_portal/usingtrafficportal.rst

```diff
@@ -296,26 +296,55 @@ Interfaces for managing the various components of Traffic Control and how they i
+Use the `Select Columns` menu to select the server columns to view and search. Columns can also be rearranged using drag-and-drop. Available server columns include:
+
+:Cache Group:		[Visible by default] The name of the :term:`Cache Group` to which this server belongs
+:CDN:				[Visible by default] The name of the CDN to which the server belongs
+:Domain:			[Visible by default] The domain part of the server's :abbr:`FQDN (Fully Qualified Domain Name)`
+:Host:				[Visible by default] The (short) hostname of the server
+:HTTPS Port:		The port on which the server listens for incoming HTTPS connections/requests
+:ID:				An integral, unique identifier for this server
+:ILO IP Address:	[Visible by default] The IPv4 address of the server's :abbr:`ILO (Integrated Lights-Out)` service
 
 	.. seealso:: `Hewlett Packard ILO Wikipedia Page <https://en.wikipedia.org/wiki/HP_Integrated_Lights-Out>`_
 
+:ILO IP Gateway:	The IPv4 gateway address of the server's :abbr:`ILO (Integrated Lights-Out)` service
+:ILO IP Netmask:	The IPv4 subnet mask of the server's :abbr:`ILO (Integrated Lights-Out)` service
+:ILO Username:		The user name for the server's :abbr:`ILO (Integrated Lights-Out)` service
+:Interface Name:	The name of the primary network interface used by the server
+:IPv6 Address:		[Visible by default] The IPv6 address and subnet mask of ``interfaceName``
+:IPv6 Gateway:		The IPv6 address of the gateway used by ``interfaceName``
+:Last Updated:		The date and time at which this server description was last modified
+:Mgmt IP Address:	The IPv4 address of some network interface on the server used for 'management'
+:Mgmt IP Gateway:	The IPv4 address of a gateway used by some network interface on the server used for 'management'
+:Mgmt IP Netmask:	The IPv4 subnet mask used by some network interface on the server used for 'management'
+:Network Gateway:	The IPv4 address of the gateway used by ``interfaceName``
+:Network IP:		[Visible by default] The IPv4 address of ``interfaceName``
+:Network MTU:		The Maximum Transmission Unit (MTU) to configured on ``interfaceName``
+:Network Subnet:	The IPv4 subnet mask used by ``interfaceName``
+:Offline Reason:	A user-entered reason why the server is in ADMIN_DOWN or OFFLINE status
+:Phys Location:		[Visible by default] The name of the physical location where the server resides
+:Profile:			[Visible by default] The :ref:`profile-name` of the :term:`Profile` used by this server
+:Rack:				A string indicating "server rack" location
+:Reval Pending:		[Visible by default] A boolean value represented as a clock (content invalidation/revalidation is pending) or green check mark (content invalidation/revalidation is not pending)
+:Router Hostname:	The human-readable name of the router responsible for reaching this server
+:Router Port Name:	The human-readable name of the port used by the router responsible for reaching this server
```

Comment:
TP doesn't allow you to set/change router hostname/port name, and honestly I have no idea what those could be. TR hostname and port... name? TO clearly doesn't actually populate that automatically, so you'd need to set it yourself and remember to update it manually on infrastructural changes.

So I guess my question is: does this need to be a column?

Reply:
well, it is a property on the server that somebody may or may not want to see so i'd rather leave it up to the user.

Reply:
hmm. i'm surprised TP doesn't allow you...actually, maybe you are right. i think i will drop them.

Reply:
you can define those values in TP on the server form

![image](https://user-images.githubusercontent.com/251272/62496469-3db8a800-b796-11e9-9484-68d92107d45c.png)

so i think it's a good idea to have them accessible in the servers table.